content/hardware/additional/electronic-switches/index.md

```diff
@@ -0,0 +1,23 @@
++++
+title = "Electronic Switches (Relays and MOSFETs)"
+description = "Enables the autopilot and/or onboard computer to control higher voltage (>5V DC) and high power circuits via software."
+date = 2022-10-11T17:33:19+11:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 35
+draft = false
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+
+Electronic switches are useful for turning higher voltage (>5V DC) auxiliary equipment on and off via digital outputs on the autopilot board. Most commonly they use relay or MOSFET components.
+
+Up to four electronic switches can be operated via joystick button functions when connected to the appropriate signal outputs.
```

Comment:
> or have another pass at these docs linking things after the great migration is done.

Yep, that's the plan. There are several other things that should be linked too, but I don't want to make a bunch of links to the old docs and then have to change them all (partly because doing so would reduce the motivation to changing them, and not doing so increases the motivation to get the other sections done sooner rather than later).